--- crypt-lib.sh	2019-09-18 16:30:43.546776135 +0200
+++ /usr/lib/dracut/modules.d/90crypt/crypt-lib.sh	2018-10-08 15:38:33.000000000 +0200
@@ -224,14 +224,6 @@
                 die "No loop file support to decrypt '$keypath' on '$keydev'."
             fi
             ;;
-        fido2)
-            if [ -f /lib/dracut-crypt-fido2-lib.sh ]; then
-                . /lib/dracut-crypt-fido2-lib.sh
-                fido2_decrypt "$mntp" "$keypath" "$keydev" "$device"
-            else
-                die "No FIDO2 support to decrypt '$keypath' on '$keydev'."
-            fi
-            ;;
         *) cat "$mntp/$keypath" ;;
     esac
 
